index.html

```diff
@@ -1,657 +1,1131 @@
+                                </div>
+                              </div>
+                              <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_1801542987.coreimg{.width}.svg/1669130241057/linkedin.svg" data-asset="/content/dam/LeaderImpact/site-./assets/linkedin.svg" data-title="LinkedIn" id="image-bb85cd5a39" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                                  <img src="./assets/linkedin.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="LinkedIn">
+                                </div>
+                              </div>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
                     </div>
+                  </div>
                 </div>
-            </div>
-            <div class="footer-bottom-container">
-                <div class="inner-wrap">
-                    <div class="copy-right">
-                        <p class="content">©1994-2022. LeaderImpact™ All Rights Reserved.</p>
-                        <img srcset="./assets/copyright-ecfa.svg" src="./assets/copyright-ecfa.svg" loading="lazy" class="logo-img" itemprop="contentUrl"  alt="logo" title="logo">
-                    </div>
-                    <div class="term-policy-wrap">
-                        <span class="term-policy-item" data-cmp-data-layer="">
-                            <a data-cmp-clickable="" class="nav-link-item" href="">Privacy</a>
-                        </span> 
-                        <span class="term-policy-item" data-cmp-data-layer="">
-                            <a data-cmp-clickable="" class="nav-link-item" href="">Term of Use</a>
-                        </span> 
+              </div>
+              <!-- FOOTER BOTTOM WRAP -->
+              <div class="container responsivegrid footer-bottom-container aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                <div id="container-48cdc20019" class="cmp-container">
+                  <!-- FOOTER COPYRIGHT -->
+                  <div class="image copy-right-img aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container_1777610207/container/container/image.coreimg{.width}.svg/1669130630947/copyright-ecfa.svg" data-asset="/content/dam/LeaderImpact/site-./assets/copyright-ecfa.svg" data-title="logo" id="image-77a3439061" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                      <p class="cmp-image__title">©1994-2022. LeaderImpact™ All Rights Reserved.</p>

```

Comment:
@Prystal254 If you're trying to use the image's caption, this isn't where it goes. Please refer to the [documentation](https://www.aemcomponents.dev/content/core-components-examples/library/core-content/image.html#tabs-853dd31146-item-cce684c9b6-tab).